.lando.yml

```diff
@@ -39,6 +39,7 @@ services:
     build:
       - echo "Installing Node.js dependencies..."
       - npm -C /app/web/themes/custom/westy install
+      - npm -C /app/web/themes/custom/westy run production
```

Comment:
I think we should also run `npm install` and `npm run production` as part of `lando refresh` on line 129. (We do a similar theme rebuild in portlandor for `lando refresh`.)